cpp/ps0002/p0002-soln-cpp.cpp

```diff
@@ -0,0 +1,40 @@
+#include <iostream>
+using namespace std;
+
+int main()
+{
+  // عدد سكان عمارة
+  short populationBuilding = 100; // 0.5/0.75
+  
+  // عدد البدائل لدوا في الصيدلية
+  short drug_Alternatives = 5; // 0.75/0.75
+  
+  // أفراد تيم في شركة برمجة
+  short team_Members = 8; // 0.75/0.75
+  
+  // سعر واحدة من مخبوزات الفرن
+  float bakery_Price = 3.44; // 0.75/0.75
+  
+  // الحرف الأول من اسم شخص
+  char first_char= 'A'; // 0.75/0.75
+  
+  // عدد المواليد في دولة تعدادها كبير زي الصين
+  int chinaBirths = 4985700; // 0.25/0.75
+  
+  // ناتج اكثر دقة لقسمة رقم على 7
+  //تنفعfloat or double
+  //بس double اكثر دقة
+  double division_Result = 10.0 / 7.0; // 0.75/0.75
+  
+  // فاتورة الكهرباء لخط انتاج في مصر
+  //تنفعfloat or double
+  double electricity_Bill = 1500.75; // 0.75/0.75
+  
+  // عمر أشخاص كبار و صغار في السن بالثانية
+  unsigned int age_Seconds = 31536000; // 0.5/0.75
+
+  // total 5.75/6.75
+   
+  return 0;
+}
+// great work 😎 9/10 ⭐

```

Comment:
- [x] program is in the correct path
- [x] no syntax errors
- [x] formatted code to be clear and readable
- [x] commented some phrases for blocks of code